File: Memory.php
<?php

class Memory extends Hardware{
    private $frekuensi;
    private $size;
    private $cuda;
    
   

    public function __construct($frekuensi = 0, $size = 0, $cuda = 0) {
        $this->frekuensi = $frekuensi;
        $this->size = $size;

    }

    public function setFrekuensi($frekuensi) { //
        $this->frekuensi = $frekuensi;
    }
    public function getFrekuensi() { 
        return $this->frekuensi; 
        
    }

    public function setSize($size) { //
        $this->size = $size;
    }

    public function getSize(){ 
        return $this->size;

    } 

    public function setCuda($cuda) { //
        $this->cuda = $cuda;
    }

    public function getCuda(){  
        return $this->cuda;

    } 

}
?>

File: Python/Memory.py
from Hardware import Hardware


class Memory(Hardware):


    def __init__(self, frekuensi= 0, size= 0, cuda= 0):
        self.frekuensi = frekuensi
        self.size = size
        self.cuda = cuda
        
    def set_frekuensi(self, frekuensi):
        self.frekuensi = frekuensi    

    def set_size(self, size):
        self.size = size

    def set_cuda(self, cuda):
        self.cuda = cuda
    
    def get_frekuensi(self):
        return self.frekuensi

    def get_size(self):
        return self.size
    def get_cuda(self):
        return self.cuda